get-started/evaluate-elastic.md

```diff
@@ -0,0 +1,336 @@
+---
+products:
+  - id: elasticsearch
+  - id: elastic-stack
+  - id: observability
+  - id: security
+applies_to:
+  serverless:
+  stack:
+description: Build a successful proof of concept during your Elastic trial. Learn how to define success criteria, choose the right deployment and use case, measure results, and prepare for production.
+---
+
+# Evaluate Elastic during a trial
+
+If you're evaluating Elastic during a trial, this guide helps you build a meaningful proof of concept (PoC) that demonstrates clear value to your organization. Rather than prescribing specific technical steps, this guide focuses on the evaluation process itself, helping you make strategic decisions and measure success.
+
+## What's included in your trial
+
+Your Elastic trial gives you full access to explore our platform's capabilities:
+
+- All features available across [Search](/solutions/search.md), [{{observability}}](/solutions/observability.md), and [Security](/solutions/security.md) solutions.
```

Comment:
I think this depends which project type you choose (i.e. you can't have them all in Serverless context)
```suggestion
- All features available across [Search](/solutions/search.md), [{{observability}}](/solutions/observability.md), and [Security](/solutions/security.md) solutions, depending on your choice of deployment and project type.
```